### DISCUSSION TITLE
Inactive (archived) asset accounts still displayed

### DISCUSSION THREAD

**Discussion by MasterZhang007** [firefly-iii/firefly-iii#7024] [General]
Hi, I have used FireflyIII for two years to manage my personal financial, but I am quite confused about something.

I disabled three asset accounts last year and before I did that, the balances were set to zero. However, I can still see them on the report page (Dashboard > NET WORTH > Account balances).

If I disable an account, it will never generate any new transactions in a financial year, so it will not be seen.

Is there any way not to display them? I look forward to your answer.

**Comment by JC5** [comment]
No. They will remain in that particular list.

**Reply by MasterZhang007** [reply]
It seems strange to me, but I respect your answer

**Reply by JC5** [reply]
You can customize reports from the /reports page, if you want to make a custom selection.

**Reply by MasterZhang007** [reply]
OK I got it, thank you!
